### ISSUE TITLE
Travis build is failing during java install phase

### ISSUE THREAD

**Issue by alex-d-bondarev** [concordion/concordion#296]
[All latest](https://travis-ci.org/concordion/concordion/builds/639097440?utm_source=github_status&utm_medium=notification) Travis CI builds are failing during java install phase:
**Java 7:**
```
Installing openjdk7
$ export JAVA_HOME=~/openjdk7
$ export PATH="$JAVA_HOME/bin:$PATH"
$ ~/bin/install-jdk.sh --target "/home/travis/openjdk7" --workspace "/home/travis/.cache/install-jdk" --feature "7" --license "GPL" --cacerts
Ignoring license option: GPL -- using GPLv2+CE by default
install-jdk.sh 2020-01-14
Expected feature release number in range of 9 to 15, but got: 7
The command "~/bin/install-jdk.sh --target "/home/travis/openjdk7" --workspace "/home/travis/.cache/install-jdk" --feature "7" --license "GPL" --cacerts" failed and exited with 3 during .
```
**Java 8:**
```
Installing oraclejdk8
$ export JAVA_HOME=~/oraclejdk8
$ export PATH="$JAVA_HOME/bin:$PATH"
$ ~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"
Ignoring license option: BCL -- using GPLv2+CE by default
install-jdk.sh 2020-01-14
Expected feature release number in range of 9 to 15, but got: 8
The command "~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"" failed and exited with 3 during .
```
**Java 9:**
```
> Task :compileJava FAILED
```

**Comment by nigelcharman** [comment]
Please merge in the trusty branch - 
https://github.com/concordion/concordion/tree/trusty

This is failing due to #294. Merging this into your PR should fix 
everything.

Thanks!

**Comment by alex-d-bondarev** [comment]
Hi @nigelcharman, that worked. The build is passing now and PR is green.

**Comment by nigelcharman** [comment]
Fixed by #295
